package.json

```diff
@@ -33,6 +33,7 @@
     "sense": "nebula sense --ext ./src/extension/ext-raw.js --meta ./src/extension/meta.json",
     "lint": "eslint src",
     "start": "nebula serve --type sn-org-chart",
+    "spec": "scriptappy-from-jsdoc -c ./spec-configs/props.conf.js",
```

Comment:
Does this need to run automatically? E.g. when version is bumped?

Reply:
I'll do that in a separate PR